### DISCUSSION TITLE
Display only emotes available for sending in Emote-Only Mode

### DISCUSSION THREAD

**Discussion by x9136** [Chatterino/chatterino2#4975] [Ideas] [answered]
If the emote list contains only emotes that can be sent in Emote-Only Mode, there will be fewer attempts to check which emotes can be sent and which can't.

Now it is difficult to make error-free choices or get used to it, because this mode is enabled in special cases.

We also have to take into account that some roles can send messages in any mode. As a result, if it is not possible to automatically define a role, the solution could be a setting where you choose which channels to hide emotes for depending on the mode (or for all channels, if you don't have any special roles anywhere).

**Comment by pajlada** [comment, marked as answer]
Since this is not behaviour that changes (i.e. what emotes you can and cannot post will stay the same for emote mode today and emote mode in a week), and it's uncommon enough (especially for the mode to be enabled for long), I don't want to implement this into Chatterino.

You as a user can post everything under `Subs` and the `Twitch` emotes under `Global` - that's it
